content/spin/v2/variables.md

```diff
@@ -119,42 +119,37 @@ fn handle_spin_example(req: Request) -> Result<impl IntoResponse> {
 
 > [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-js-sdk/variables/Config.html)
 
-The function is available on [the `Config` object](https://fermyon.github.io/spin-js-sdk/variables/Config.html) and is named [`get`](https://fermyon.github.io/spin-js-sdk/interfaces/_internal_.SpinConfig.html#get).
-
-> Note that the name is `Config` rather than `Variables`.
+> Note that the prefix/suffix is `config` rather than `variables`.
```

Comment:
This is no longer the case, right?

Reply:
That is the JS part but I updated it by mistake and did not ever to the original statement but will fix now.